kit/ChildSession.cpp

```diff
@@ -3146,6 +3221,46 @@ void ChildSession::loKitCallback(const int type, const std::string& payload)
         break;
     case LOK_CALLBACK_EXPORT_FILE:
     {
+        bool isAbort = payload == "ABORT";
+        bool isError = payload == "ERROR";
+        bool isPending = payload == "PENDING";
+        bool exportWasRequested = !_exportAsWopiUrl.empty();
+
+        if (isPending) // local save has been started
+        {
+            sendTextFrame("blockui: ");
+            return;
+        }
+
+        sendTextFrame("unblockui: ");
```

Comment:
Doing this unconditionally may be problematic. We should match "blockui:" with "unblockui:" and not issue more or less of either one. Here we are issuing move "unblockui:" than we should (for example, or regular export we will be sending "unblockui:").